python/py-subprocess-tee/Portfile

```diff
@@ -17,12 +17,16 @@ long_description    This package provides a drop-in alternative to \
     subprocess.run that captures the output while still printing it in \
     real-time, just the way tee does.
 homepage            https://github.com/pycontribs/subprocess-tee
+master_sites        pypi:s/subprocess-tee
+distname            subprocess_tee-${version}
+distfiles           subprocess_tee-${version}.tar.gz
+dist_subdir         subprocess_tee-${version}
```

Comment:
of the four lines above only `distname` should be needed; please remove the others